packages/components/pie-link/README.md

```diff
@@ -57,9 +57,15 @@ import { PieLink } from '@justeattakeaway/pie-link/dist/react';
 
 ## Props
 
-| Property | Type | Default | Description |
-|---|---|---|---|
-| - | - | - | - |
+| Property     | Type        | Default       | Description                                                                 |
+| ------------ | ----------- | ------------- | --------------------------------------------------------------------------- |
+| variant      | `String`  | `default`   | Variant of the link, one of variants – default, high-visibility or inverse |
+| size         | `String`  | `medium`    | Size of the link, one of `sizes` – `medium`, `small`                 |
+| href         | `String`  | `undefined` | Native html `href` attribute                                              |
+| rel          | `String`  | `undefined` | native html `rel` attribute                                               |
```

Comment:
nit, sorry. But can all of these start with a capital letter please? Sorry

Reply:
@FayeCarter do you mean the descriptions? I agree if so but I think we should keep props camel case

Reply:
fixed